src/motion_input_shaping/shaping_comparison_test.py

```diff
@@ -26,6 +27,7 @@
 SCOPE_TIMEBASE = 1  # The scope sampling period in ms. Adjust if the whole move isn't captured.
 RESONANT_FREQUENCY = 5.07  # Input shaping resonant frequency in Hz.
 DAMPING_RATIO = 0.1  # Input shaping damping ratio.
+SHAPER_CONFIG = ShaperConfig(ShaperMode.DECEL)
```

Comment:
Might want to add a comment here. We want users to know how to use this class without having to look at the source code.

Reply:
Shaper config has been removed so this is no longer necessary